x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/integration/MlDistributedFailureIT.java

```diff
@@ -301,15 +284,9 @@ public void testCloseUnassignedFailedJobAndStopUnassignedStoppingDatafeed() thro
     public void testStopAndForceStopDatafeed() throws Exception {
         internalCluster().ensureAtMostNumDataNodes(0);
         logger.info("Starting dedicated master node...");
-        internalCluster().startNode(Settings.builder()
-            .put("node.master", true)
-            .put("node.data", true)
-            .put("node.ml", false)
-            .build());
+        internalCluster().startMasterOnlyNode();
```

Comment:
The code used to enable `master` and `data` here, was that a mistake in the old code?

Reply:
Yeah, that was unnecessary in this test.